Formula/aqbanking.rb

```diff
@@ -19,24 +19,23 @@ class Aqbanking < Formula
     depends_on "libtool" => :build
   end
 
-  depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "gmp"
   depends_on "gwenhywfar"
   depends_on "ktoblzcheck"
   depends_on "libxml2"
   depends_on "libxmlsec1"
   depends_on "libxslt"
+  depends_on "pkg-config"
```

Comment:
maybe `depends_on "pkg-config" => :build`?

Reply:
If it has to be runtime dependency, we might need to specify as comment.

Reply:
Done ✅ Also reverted the minor whitespace change